composer.json

```diff
@@ -35,8 +35,9 @@
 		"phpcompatibility/php-compatibility": "*",
 		"phpstan/phpstan-shim": "^0.9.2",
 		"keboola/table-backend-utils": "^0.7",
-		"brianium/paratest": "2.*"
-	},
+		"brianium/paratest": "2.*",
+        "ext-pdo": "*"
```

Comment:
toto tu imho nemusi byt nie ? Len si tu metodu presunul do traity tak to uz fungovalo nie ?

Reply:
fungovalo, jen storm nadava a imho bychom to tam specifikovane mit meli

Reply:
Hm divne mozno mas nieco inac nastavene v PHPstorme pretoze mne nic nesvieti.